Over21/views/AgeIndicatorView.swift

```diff
@@ -32,6 +32,18 @@ class AgeIndicatorView: UIView {
         
         return lbl
     }()
+    public var printerConnectionLabel: UILabel = {
+        let lbl = UILabel()
+        lbl.translatesAutoresizingMaskIntoConstraints = false
+        lbl.font = UIFont.systemFont(ofSize: 24, weight: UIFont.Weight.medium)
+        //lbl.adjustsFontSizeToFitWidth = true
```

Comment:
we should remove this commented out code.

Reply:
removed comment line